### PR TITLE
[Documentation] seriesuid2fileid

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Generated using:
pod2markdown tools/seriesuid2fileid docs/scripts_md/tools/seriesuid2fileid.md

User can type:
perldoc tools/tools/seriesuid2fileid
to get the documentation at the terminal

### CODE REVIEW

File: tools/seriesuid2fileid [hunk 1]
```diff
@@ -1,4 +1,37 @@
...
+
+=head1 NAME
+
+seriesuid2fileid -- a script that displays relevant information and tracks the
+status of an acquired imaging series based on the series UID as an input
+argument (provided as a STDIN).
+
+=head1 SYNOPSIS
+
+perl tools/seriesuid2fileid
```
Comment (cmadjar) [inline tools/seriesuid2fileid]: Shouldn't the script be called seriesuid2fileid.pl? Missing the .pl extension. Maybe you can fix this in this PR?

File: tools/seriesuid2fileid [hunk 2]
```diff
@@ -1,4 +1,37 @@
 #!/usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+seriesuid2fileid -- a script that displays relevant information and tracks the
+status of an acquired imaging series based on the series UID as an input
+argument (provided as a STDIN).
```
Comment (cmadjar) [inline tools/seriesuid2fileid]: Maybe rephrase the tiny description up there to something like:

"displays a report (including violation status) about imaging dataset(s) based on Series UID(s) given as STDIN"

File: tools/seriesuid2fileid (hunk 1 above)
Comment (MounaSafiHarab) [reply tools/seriesuid2fileid]: as agreed, we will do this separately (RM#13755)

File: tools/seriesuid2fileid (hunk 2 above)
Comment (MounaSafiHarab) [reply tools/seriesuid2fileid]: rephrased (mostly like you suggested with a little bit of difference); see latest commit and let me know if u disagree

**Comment by MounaSafiHarab** [comment]
@cmadjar 
changes addressed!

File: docs/scripts_md/seriesuid2fileid.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+seriesuid2fileid -- a script that displays a report about the pipeline insertion
+progress and outcome (including MRI violation status) of imaging datasets, based
+on series UID(s) provided as STDIN.
+
+# SYNOPSIS
+
+perl tools/seriesuid2fileid
+
+There are no available options. Once the script is invoked, series UID can be
+input to display the status. The $profile file for database connection
```
Comment (cmadjar) [inline docs/scripts_md/seriesuid2fileid.md]: either replace $profile by profile or put $profile in code format

File: docs/scripts_md/seriesuid2fileid.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+# DESCRIPTION
+
+The program takes series UID from STDIN and returns a report with:
+
+    - SeriesUID
+    - SeriesDescription
+    - TarchiveID
+    - m_p_v_s_ID
+    - mri_v_log
```
Comment (cmadjar) [inline docs/scripts_md/seriesuid2fileid.md]: Might be good to explain what m_p_v_s_ID and mri_v_log stand for

File: docs/scripts_md/seriesuid2fileid.md (hunk 3 above)
Comment (MounaSafiHarab) [reply docs/scripts_md/seriesuid2fileid.md]: ok

File: docs/scripts_md/seriesuid2fileid.md (hunk 4 above)
Comment (MounaSafiHarab) [reply docs/scripts_md/seriesuid2fileid.md]: good idea.

Comment (MounaSafiHarab) [reply docs/scripts_md/seriesuid2fileid.md]: done
